src/script.ts

```diff
@@ -24,19 +24,19 @@ export class Script {
 
     copy(pathFrom: string, pathTo: string) {
         if (this.isWindows) {
-            this.command(`copy /Y "${pathFrom}" "${pathTo}"`);
+            this.command(`copy /Y "${this.folder}\\${pathFrom}" "${pathTo}"`);
         }
         else {
-            this.command(`cp "${pathFrom}" "${pathTo}"`);
+            this.command(`cp "${this.folder}\\${pathFrom}" "${pathTo}"`);
```

Comment:
This doesn't seem like correct path separator for unix platforms.

Reply:
Could use "/" for the else statements instead.